client/ayon_blender/api/plugin.py

```diff
@@ -254,6 +254,8 @@ def create(
         instances = bpy.data.collections.get(AYON_INSTANCES)
         if not instances:
             instances = bpy.data.collections.new(name=AYON_INSTANCES)
+            instances.color_tag = "COLOR_04"
+            instances.use_fake_user = True
```

Comment:
The fake user is redundant here, right? because we link it to the scene collection? Or are we doing this just to ensure it's more persistent? or?

Reply:
I am thinking if we should just make the color_tag or fake users for newly created ayon instances only